src/agents/agent.py

```diff
@@ -17,12 +17,13 @@
 
 from src.bert.sentence import SentenceBert
 from src.memory import KnowledgeBase
-from src.browser.search import BingSearch
+from src.browser.search import *
```

Comment:
@its-arun it's best to perform explicit imports as it prevents possible variable overlap.